knip.jsonc

```diff
@@ -0,0 +1,46 @@
+{
+    "workspaces": {
+        ".": {
+            // These entries are complementary to the ones found in package.json
+            "entry": [
+                "lib/rules/index.js",
+                "tools/internal-rules/index.js",
+                "tools/update-rule-types.js",
+                "wdio.conf.js" // Remove when Knip has a wdio plugin
+            ],
+            "project": ["{conf,lib,tools}/**/*.js"],
+            "mocha": {
+                "entry": [
+                    "tests/{bin,conf,lib,tools}/**/*.js", // see Makefile.js
+                    "tests/_utils/test-lazy-loading-rules.js"
+                ],
```

Comment:
[Question] IIRC from our pairing this ideally have been discoverable from the built-in Knip Mocha logic but wasn't. Is that the case? Can you speak more to this?

Reply:
The comment refers to `Makefile.js`, which has custom logic related to this. So unfortunately those entry paths can't be inferred by the Mocha plugin in Knip.